kong/plugins/runscope-log/log.lua

```diff
@@ -0,0 +1,85 @@
+local cjson = require "cjson"
+local url = require "socket.url"
+
+local _M = {}
+
+local HTTPS = "https"
+
+-- Generates http payload .
+-- @param `method` http method to be used to send data
+-- @param `parsed_url` contains the host details
+-- @param `message`  Message to be logged
+-- @return `payload` http payload
+local function generate_post_payload(parsed_url, access_token, message)
+  local body = cjson.encode(message)
+  local payload = string.format(
```

Comment:
It is best to cache this method (`string.format`) at the module level to avoid an access to the globals since it will be executed very often.

```lua
local string_format = string.format
-- ...
local payload = string_format(...)
```